buildSrc/src/main/groovy/io.micronaut.build.internal.security-tests.gradle

```
@@ -9,3 +9,10 @@ dependencies {
     testImplementation(platform("io.micronaut.platform:micronaut-platform:${libs.versions.micronaut.platform.get()}"))
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
+
+//TODO remove once Micronaut Test ships Spock version compatible with Groovy 5
+configurations.all {
+    resolutionStrategy {
+        force("org.spockframework:spock-core:2.4-M7-groovy-5.0")
```

```suggestion
        force("org.spockframework:spock-core:2.4-groovy-5.0")
```

there is a release already a stable release of of spock compatible with Groovy 5